databricks_cli/execution_context/cli.py

```diff
@@ -0,0 +1,251 @@
+              help=ClusterIdClickType.help)
+@click.option('--context-id', required=True, type=ExecutionContextIdClickType(),
+              help=ExecutionContextIdClickType.help)
+@click.option('--command', required=True, type=CommandStringType(),
+              help=CommandStringType.help)
+@click.option('--wait', required=False, default=False, type=click.BOOL,
+              help="If true wait for command completion, otherwise schedule "
+                   "the command and return immediately.")
+@click.option('--language', required=False, default="python", 
+              type=click.Choice(['python', 'scala', 'sql'], case_sensitive=False),
+              help="The language for the context.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def execute_command_cli(api_client, output, cluster_id, context_id, command, language, wait): #  NOQA
+    """
+    Executes a command within an existing execution context.
+    """
+    client = ExecutionContextApi(api_client)
+    result = client.execute_command(cluster_id, context_id, command, language)
+    if not wait:
+        click.echo(pretty_format(result))
+        return
+
+    command_id = result["id"]
+    _wait_for_command(client, output, cluster_id, context_id, command_id)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help="Executes a single command within a new execution context.")
+@click.option('--output', default=None, help=CommandOutputType.help,
+              type=CommandOutputType())
+@click.option('--cluster-id', required=True, type=ClusterIdClickType(),
+              help=ClusterIdClickType.help)
+@click.option('--command', required=True, type=CommandStringType(),
+              help=CommandStringType.help)
+@click.option('--language', required=False, default="python", 
+              type=click.Choice(['python', 'scala', 'sql'], case_sensitive=False),
+              help="The language for the context.")
+@click.option('--wait', required=False, default=False, type=click.BOOL,
+              help="If true wait for command completion, otherwise schedule"
+                   " the command and return immediately.")
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def execute_command_once_cli(api_client, output, cluster_id, command, language, wait): #  NOQA
+    """
+    Executes a single command within a new execution context.
+    The execution context is automatically created and cleaned up.
+    """
+    client = ExecutionContextApi(api_client)
+
+    with ExecutionContext(api_client, cluster_id) as context:
+        result = client.execute_command(cluster_id, context.id, command, language)
+        if not wait:
+            click.echo(pretty_format(result))
+            return
+
+        command_id = result["id"]
+        _wait_for_command(client, output, cluster_id, context.id, command_id)
+
+
+def _wait_for_command(client, output, cluster_id, context_id, command_id):
```

Comment:
and what would happen if user executes `while True: time.sleep(1)` as the command? All retryable executions must have a timeout specified. 20 minutes is a reasonable default timeout. Is must be possible to override that timeout by user controlled parameter.